static/app/views/insights/crons/components/monitorCheckIns.tsx

```diff
@@ -198,10 +204,51 @@ export function MonitorCheckIns({monitor, monitorEnvs}: Props) {
   );
 }
 
+interface TimeoutLateByProps {
+  duration: number;
+  monitor: Monitor;
+}
+
+function TimeoutLateBy({monitor, duration}: TimeoutLateByProps) {
+  const maxRuntimeSeconds = (monitor.config.max_runtime ?? DEFAULT_MAX_RUNTIME) * 60;
+  const lateBySecond = duration / 1000 - maxRuntimeSeconds;
+
+  const maxRuntime = (
+    <strong>
+      <Duration seconds={(monitor.config.max_runtime ?? DEFAULT_MAX_RUNTIME) * 60} />
+    </strong>
+  );
+
+  const lateBy = (
+    <strong>
+      <Duration seconds={lateBySecond} />
+    </strong>
+  );
+
+  return (
+    <Tooltip
+      title={tct(
+        'The final check-in for this job occurred [lateBy] after it was marked as timed out. The configured maximum allowed runtime is [maxRuntime].',
```

Comment:
```suggestion
        'The closing check-in occurred [lateBy] after it was marked as timed out. The configured maximum allowed runtime is [maxRuntime].',
```

Reply:
removed referenced to job just because we don't say that in the UI right now (and keeps the messaging shorter)

Reply:
`it` doesn't make as much sense now. maybe "after this check-in was marked as timed out"

Reply:
or we could say "after the in-prgoress check-in was marked as timed out"